CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 # Changelog
 
+## v1.38.1 (2025-01-20)
+
+    Fix TOCTOU error when bind and cache mounts use "src" values
```

Comment:
Any possibility to adding a `[release-1.38]` tag to these commits?  It looks like the push just grabbed them from main.

Reply:
They were cherry-picked for their respective branches.  For release-1.38 I don't recall needing to make any changes.  I'm not sure I understand the benefit of going back and retitling the commits, though...?

Reply:
Oh, the actual commits were merged, so changing their commit logs would mean rewriting the history on the branch, which means we can't do that anyway.

Reply:
I just find it easier when sorting through history and the change logs to see the leading tags.  YMMV, I'm OK with not adding them

Reply:
Right now, they match the commits.  It's fine.